src/coreclr/clr.featuredefines.props

```diff
@@ -1,7 +1,6 @@
 <Project>
     <PropertyGroup>
         <FeatureCoreCLR>true</FeatureCoreCLR>
-        <FeatureEventTrace>true</FeatureEventTrace>
```

Comment:
was it intended to drop the default for FeatureEvetnTrace here ?

Reply:
Yes, as far as I know, this property is unused anywhere except for defining `FEATURE_EVENT_TRACE` in managed code, and then `FEATURE_EVENT_TRACE` is unused anywhere in managed code.